translate.sh

```diff
@@ -1,3 +1,6 @@
-#!/bin/bash
-j2objc -d src/main/objc -use-arc -cp build/libs/lucenestudy.jar `find src/main/java -name *.java`
-j2objc -d src/test/objc -use-arc -cp src/test/java:build/libs/lucenestudy.jar:$J2OBJC_ROOT/lib/j2objc_junit.jar:$J2OBJC_ROOT/lib/hamcrest-core-1.3.jar `find src/test/java -name *.java`
+#!/bin/bash -e
+
+J2OBJC=./vendor/j2objc/j2objc
+
+${J2OBJC} -d src/main/objc -use-arc -cp build/libs/lucenestudy.jar `find src/main/java -name *.java`
+${J2OBJC} -d src/test/objc -use-arc -cp src/test/java:build/libs/lucenestudy.jar:${J2OBJC}/lib/j2objc_junit.jar:${J2OBJC}/lib/hamcrest-core-1.3.jar `find src/test/java -name *.java`
```

Comment:
The reason why tests failed to translate is because of the classpath; it needs to be:

    -cp ./vendor/j2objc/lib/j2objc_junit.jar:...

But because `$J2OBJC` is currently defined as `./vendor/j2objc/j2objc`, this is expanded to:

    -cp ./vendor/j2objc/j2objc/lib/j2objc_junit.jar:...

So if this is changed to:

    J2OBJC_ROOT=./vendor/j2objc
    ${J2OBJC_ROOT}/j2objc -d ...
    ${J2OBJC_ROOT}/j2objc -d ... -cp src/test/java:build/libs/lucenestudy.jar:${J2OBJC_ROOT}/lib/j2objc_junit.jar:...

The tests will be translated.